### DISCUSSION TITLE
Qubino Dimmer: Dim and/or On/Off!?

### DISCUSSION THREAD

**Discussion by hestiap** [zwave-js/zwave-js-ui#2026] [Q&A]
Hi,
I have a Qubino Flush Dimmer Plus  ZMNHDD
![image](https://user-images.githubusercontent.com/73014953/144767592-73feb3fe-444f-474e-9dc3-b44606006bff.png)
and in dz it's shown as a dimmer and a switch (On/Off)
![image](https://user-images.githubusercontent.com/73014953/144767648-6b79be01-d12b-49d0-a46e-7a1c40642968.png)

When I switched the "real" switch (in the wall), the dimmer in dz was set to a level, and the switch On/Off in dz stayed off
2021-12-05 23:14:18.999 INFO ZWAVE: Node 17: value updated: 38-0-currentValue 0 => 99
2021-12-05 23:14:20.026 INFO ZWAVE: Node 17: value updated: 50-0-value-66049 0 => 48.4

When I switched the dimmer in dz, the light went on, in dz the switch On/Off stayed off
2021-12-05 23:23:52.542 INFO MQTT: Message received on zwave/17/38/0/targetValue/set: '47'
2021-12-05 23:23:52.544 INFO ZWAVE: Writing 47 to 17-38-0-targetValue

When I switched the switch On/Off in dz, the light went on, in dz the dimmer stayed off
2021-12-05 23:21:12.488 INFO MQTT: Message received on zwave/17/37/0/targetValue/set: 'true'
2021-12-05 23:21:12.491 INFO ZWAVE: Writing true to 17-37-0-targetValue
2021-12-05 23:21:12.739 INFO ZWAVE: Node 17: value updated: 37-0-currentValue false => true
2021-12-05 23:21:14.113 INFO ZWAVE: Node 17: value updated: 37-0-currentValue true => true

When I switched the "real" switch (in the wall) to Off, nothing happened in dz = the switch On/Off in dz stayed On!
2021-12-05 23:22:50.394 INFO ZWAVE: Node 17: value updated: 38-0-currentValue 0 => 0

For me, it is not coherent, If there is a switch and a dimmer both should be updated

**Comment by hestiap** [comment]
do I have to open an issue on this problem?

**Reply by robertsLando** [reply]
@hestiap Sorry I missed this, BTW as explained in docs I don't have any plan right now to keep mqtt discovery updated

**Reply by hestiap** [reply]
@robertsLando Thanks for your reply
sorry, I didn't read the docs enough, and didn't know about "mqtt discovery updated " will not be updated.
I don't think that what I've reported is only relative to mqtt discovery, perhaps more about mqtt only? But I'm not an expert of both scope!
I've noticed that when I switched the "real switch" there were no mqtt messages for the "zwavejs2mqtt On/Off switch", but on the other direction mqtt to zwavejs2mqtt it worked. That's "only" a message to add?

**Reply by robertsLando** [reply]
> I've noticed that when I switched the "real switch" there were no mqtt messages for the "zwavejs2mqtt On/Off switch"

This is a problem that usually happens when devices are not reporting changes. Could be a device fw issue or something that could be fixed from device OCnfiguration CC

**Reply by hestiap** [reply]
@robertsLando
thanks for your reply
"from device Configuration CC"
could you give me some info in this, please? an example?

**Reply by robertsLando** [reply]
maybe @blhoward2 can help here

**Comment by LordMike** [comment]
On the same topic, I have a bunch of Logic Group ZDB5100 Matrix switches.. They have binary switch and multilevel switch capabilities (on/off + dimmers).

I've noticed that:

* When I set binary switch off, multilevel _does not_ change
* When I set multilevel to 0, binary switch _does not_ change
* When I've set the binary to off, and then turn on the device using the multilevel commands (brightness=100% f.ex.), the binary _does not_ change

Basically - I do not receive Report's for stuff that I didn't send commands to. So there is a natural state issue on my end, where the two settings don't follow along.

I've asked the vendor if this "is normal", or what.. I imagine that it's deliberate, as you would otherwise have 2, 3 or 4 times the messages on the network.

**Reply by hestiap** [reply]
I've added in GROUPS I1, I2, I3 basic on/off 
![image](https://user-images.githubusercontent.com/73014953/147393296-ef15f088-20c8-486d-8e9d-d73e782a7cdd.png)
no change: nothing is sent regarding the basic on/off
```
...
2021-12-25 21:33:14.279 INFO ZWAVE: Node 17: value added 17-32-0-currentValue => 0
2021-12-25 21:33:14.282 INFO ZWAVE: Node 17: value updated: 32-0-currentValue 255 => 0
...
2021-12-25 21:33:14.699 INFO ZWAVE: Node 17: value updated: 32-0-currentValue 0 => 255
...
2021-12-25 21:33:16.480 INFO ZWAVE: Node 17: value updated: 38-0-currentValue 0 => 99
2021-12-25 21:33:17.528 INFO ZWAVE: Node 17: value updated: 50-0-value-66049 0 => 48.1
...
...
2021-12-25 21:33:47.797 INFO ZWAVE: Node 17: value updated: 32-0-currentValue 255 => 0
2021-12-25 21:33:49.694 INFO ZWAVE: Node 17: value updated: 38-0-currentValue 99 => 0
2021-12-25 21:33:50.742 INFO ZWAVE: Node 17: value updated: 50-0-value-66049 48.1 => 0
```
